stock_financial_risk/models/stock.py

```diff
@@ -17,10 +17,12 @@ def _action_done(self, cancel_backorder=False):
             )
             if moves:
                 raise exceptions.UserError(
-                    _("Financial risk exceeded in partner:\n%s")
-                    % moves.mapped("partner_id.name")
+                    _(
+                        "Financial risk exceeded in partner:\n%s",
+                        moves.mapped("partner_id.name"),
```

Comment:
This have to be outside of `_()`, or no translation will be done.

Reply:
@pedrobaeza could you recheck please. IIUC v14.0 [supports](https://www.odoo.com/documentation/14.0/reference/translations.html#variables) passing arguments to translation lookup.

Reply:
OK, I see, thanks for the pointer.